cartridge-extensions-scm-1.rockspec

```diff
@@ -24,5 +24,6 @@ build = {
     type = 'builtin',
     modules = {
         ['extensions'] = 'extensions.lua',
+        ['cartridge-extensions-version'] = 'cartridge-extensions-version.lua',
```

Comment:
It seems that extending the module with `extentions/*` are not expected

https://github.com/tarantool/cartridge-extensions/blob/master/README.md#how-to-export-my-functions

Let's just add a line to a `extentions.lua` then. It is not common for a package to have two upper-level script files.

Reply:
Done